### PR TITLE
Use ActiveRecord::VERSION::String to check versions

### PR DESCRIPTION
**Pull request by volkanunsal**

### CODE REVIEW

**Comment by nickyleach** [comment]
Thanks @volkanunsal! It looks like @afine took a slightly different approach in #51. I apologize for instigating duplicative work. Your perspective would certainly appreciated to make sure the changes in #51 still match your expectations in terms of behavior. If those changes look 👌 maybe we can close this PR?

**Comment by volkanunsal** [comment]
I made the changes I mentioned in the PR.

File: spec/sql_translator_spec.rb [hunk 1]
```diff
@@ -23,10 +23,9 @@
   end
 
   it 'encodes attributes not backed by a database column correctly' do
-    if Gem.loaded_specs["activerecord"].version < Gem::Version.new("4.2.1")
+    if ActiveRecord::VERSION::STRING < "4.2.1"
       skip "the attributes API was included in rails starting in 4.2.1"
-    elsif Gem.loaded_specs["activerecord"].version >= Gem::Version.new("4.2.1") &&
-        Gem.loaded_specs["activerecord"].version < Gem::Version.new("5.0.0")
+    elsif ActiveRecord::VERSION::STRING >= "4.2.1" && ActiveRecord::VERSION::STRING < "5.0.0"
```
Comment (jakeonfire) [inline spec/sql_translator_spec.rb:28]: comparing strings for version numbers could cause issues. "4.2.2" `>` "4.2.19". of course given the versions available, what you have here would work fine, but it is not best practice.

**Comment by jakeonfire** [comment]
closing this because of inactivity

**Pull request closed without merging**